Add yu.tf2pickup.eu

makemake-kbo opened the pull request:
Added tf2pickup.rs branding. Logo is a WIP and will be changed later.

garrappachc commented on src/environments/tf2pickup.rs.ts:
```diff
@@ -0,0 +1,21 @@
+    { name: 'discord', target: discordInvitation },
+    { name: 'changelog', target: 'https://github.com/makemake-kbo/client/blob/master/CHANGELOG.md' },
+  ],
+  titleSuffix: 'tf2pickup.rs • Ex-Yu Team Fortress 2 Pickup-up games',
```
Typo: Pickup-up

garrappachc reviewed (changes requested):
The `tf2pickup_logo_white.png` file should be two times smaller (in each dimension) than `tf2pickup_logo_white@2x.png`. The `@2` version is picked up on high-density displays (4k monitors, mobile devices, etc.).

makemake-kbo commented:
I updated the domain and fixed the typos. Im waiting for bobair to do the graphics so we should wait for her until we merge.
How about a version bump aswell?

makemake-kbo commented:
I removed them now. Thanks for the heads up!
